src/boot/efi/boot.c

```diff
@@ -1004,6 +1004,16 @@ static bool menu_run(
                                 status = xstrdup16(u"Reboot into firmware interface not supported.");
                         break;
 
+                case KEYPRESS(0, 0, 'O'):
+                case KEYPRESS(0, 0, 'o'):
+                        RT->ResetSystem(EfiResetShutdown, EFI_SUCCESS, 0, NULL);
+                        break;
+
+                case KEYPRESS(0, 0, 'B'):
+                case KEYPRESS(0, 0, 'b'):
```

Comment:
Hmm, so we apparently distinguish uppercase and lowercase, because we use that for resolution and other things. So maybe we shouldn't add `B` and `O` here, so that we can use them for other things in the future.

Reply:
there's prior art in e/E, d/D, h/H, k/K, j/J, p/P, f/F being entirely equivalent in the boot menu.

We do the lower/upper thing for switching between things for r/R and t/T only.

But tbh, I kinda like @medhefgo's idea of hiding this behind ctrl, in which case the uppercase thing doesn#t make sense anymore

Reply:
(the ctrl thing didn#t work, see below)